# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/PServiceWorkerContainer.ipdl
# Commit: 244df402d6eb
# Full Hash: 244df402d6ebc15d6845c4144245511021408835
# Author: Andrew Sutherland <asutherland@asutherland.org>
# Date: 2024-10-24 09:44:34
# Regressor Bug: 1131324
# File Overlap Count: 1
# Description:
#   Bug 1131324 - Expose ServiceWorkerContainer on WorkerNavigator. r=dom-worker-reviewers,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213725
# ==============================================================================

diff -r 57f70ba5f27e -r 244df402d6eb dom/serviceworkers/PServiceWorkerContainer.ipdl
--- a/dom/serviceworkers/PServiceWorkerContainer.ipdl	Thu Oct 24 03:02:39 2024 +0000
+++ b/dom/serviceworkers/PServiceWorkerContainer.ipdl	Thu Oct 24 03:02:40 2024 +0000
@@ -18,8 +18,6 @@
   manager PBackground;
 
 parent:
-  async Teardown();
-
   async Register(IPCClientInfo aClientInfo, nsCString aScopeURL, nsCString aScriptURL,
                  ServiceWorkerUpdateViaCache aUpdateViaCache)
         returns (IPCServiceWorkerRegistrationDescriptorOrCopyableErrorResult aResult);
@@ -33,7 +31,6 @@
   async GetReady(IPCClientInfo aClientInfo)
         returns (IPCServiceWorkerRegistrationDescriptorOrCopyableErrorResult aResult);
 
-child:
   async __delete__();
 };
 